practice/21.py

```diff
@@ -1,47 +1,52 @@
-"""
```

Comment:
Try to only commit files related to the given problem in this branch. Don't worry about fixing this, but note for the future.